target/linux/ath79/image/generic.mk

```diff
@@ -88,6 +88,15 @@ define Device/seama
   SEAMA_SIGNATURE :=
 endef
 
+define Device/telco_electronics_tel-t1
```

Comment:
This needs rename, too:
telco-electronics_t1
Same at the end of the block ...

And you need to update DTS file name as well.